crypto/x509/x509_vfy.c

```diff
@@ -22,7 +22,7 @@
 #include <openssl/x509v3.h>
 #include <openssl/objects.h>
 #include "internal/dane.h"
```

Comment:
There are a few files like this that probably should be moved to internal/crypto

Reply:
Working on it, see https://github.com/openssl/openssl/pull/9333#issuecomment-511147637.